happi/tests/test_client.py

```diff
@@ -226,3 +226,14 @@ def test_from_cfg(happi_cfg):
     client = Client.from_config()
     assert isinstance(client.backend, JSONBackend)
     assert client.backend.path == 'db.json'
+
+
+def test_list_choices(mockjsonclient):
```

Comment:
You can probably just use the `happi_client` fixture instead of just the `mockjsonclient`

Reply:
Changed in 2191d86